preload.js

```diff
@@ -70,6 +70,9 @@
   window.libphonenumber.PhoneNumberFormat = require('google-libphonenumber').PhoneNumberFormat;
   window.nodeNotifier = require('node-notifier');
 
+  window.loadImage = require('blueimp-load-image');
+  window.autoOrientImage = require('./js/modules/autoOrientImage').autoOrientImage;
```

Comment:
This is how easily we can consume new CommonJS modules from `js/modules`. As for why I used a named export for a single function module, check out: https://basarat.gitbooks.io/typescript/docs/tips/defaultIsBad.html

Happy to discuss 😄

Reply:
I like the points made in that article, but without destructuring it's a whole lot of duplication. Maybe we can start destructuring here.

Reply:
👍